NEWS.md

```diff
@@ -1,7 +1,10 @@
 # admiral (development version)
 
-## New Features
 
+
+
+## New Features
+- Added helper functions to `derive_vars_merged()` (`get_flagged_records()`) and `derive_vars_query()` (`get_vars_query()`) so they call the new function to perform previous functionality exactly as it was, but with more explicit metadata resulting (#2349)
```

Comment:
@imccay13 Perhaps change to 
```
- Added helper functions to `derive_vars_merged()` (`get_flagged_records()`) and `derive_vars_query()` (`get_vars_query()`) so that those can be called independently as per user's request. derive_vars_merged function call results are not impacted by the change.
```